content/basics/index.md

```diff
@@ -131,14 +129,14 @@ in the program.
 The `let` prefix is slightly more flexible in that it can be reassigned, and should be used when `const` cannot.
 
 The use of `var` no longer makes sense in modern JavaScript; it's like `let` in that it can be reassigned,
-but is also 'hoisted' to the top of the current code block.
+but is also ['hoisted'](https://developer.mozilla.org/en-US/docs/Glossary/Hoisting) to the top of the current function scope.
 
 Given this is a common source of bugs, and given it offers no real advantage over `let`, you should really always use `let`.
 
 ```js
-const myVar      = 10;
```

Comment:
Probably best  to avoid weird indentation things in code examples. Don't want to set a precedent.